pkgs/top-level/pidgin-packages.nix

```diff
@@ -0,0 +1,70 @@
+{ lib
+, newScope
+, pidgin
+, texlive
+, config
+}:
+
+lib.makeScope newScope (self:
+  let callPackage = self.callPackage;
+  in {
+    inherit callPackage;
```

Comment:
```suggestion
```

no need to inherit callPackage, It's in the scope by default https://github.com/NixOS/nixpkgs/blob/5daba3aa35e8264613298e517b87c0d49e4e70c7/lib/customisation.nix#L271

Reply:
done, #til